lib/common/colors.dart

```diff
@@ -26,4 +27,7 @@ class Colors {
   static Color get commentInfoTextColor => _mediumGray;
   static Color get commentFieldColor => _lightWarmGray;
   static Color get commentFieldHintColor => _mediumGray;
+
+  // list pixel
+  static Color get listPixelBackGround => _listPixel;
```

Comment:
I think `Background` is a single word (no need for upper casing the **g** in ground.

Reply:
exactly